app/grandchallenge/workstations/views.py

```diff
@@ -75,6 +75,24 @@ def keep_alive(self, *_, **__):
                 status=HTTP_400_BAD_REQUEST,
             )
 
+    @action(detail=False)
+    def active_sessions(self, request):
```

Comment:
I don't think this needs a new endpoint, can't it be achieved by only adding the status to a django filter on the list endpoint?